src/index.ts

```diff
@@ -8,7 +8,7 @@ import * as querystring from 'querystring';
 
 const readFile = pify(fs.readFile);
 
-const GOOGLE_TOKEN_URL = 'https://accounts.google.com/o/oauth2/token';
+const GOOGLE_TOKEN_URL = 'https://www.googleapis.com/oauth2/v4/token';
```

Comment:
Is this new URL equivalent to the old one? If not, should this be a semver major change?

Reply:
As far as I can tell it's largely equivalent.  This is just the URL that's documented all over the official documentation.  Totally open to making this semver major though.

Reply:
I'm fine if it's equivalent :)